### PR TITLE
Adding Azure Pipelines CI builds

### PR DESCRIPTION
**Pull request by allydurks**
These are synced the [PartsUnlimitedMRPmicro Azure DevOps Builds](https://dev.azure.com/csebostoncrew/PartsUnlimitedMRPmicro).

### CODE REVIEW

**Comment by allydurks** [comment]
These commits should be squashed.

File: Web/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - azure-pipelines
```
Comment (allydurks) [inline Web/azure-pipelines.yml]:
```suggestion
    - master
```

File: ShipmentSrvc/azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - azure-pipelines
```
Comment (allydurks) [inline ShipmentSrvc/azure-pipelines.yml]:
```suggestion
    - master
```

File: QuoteSrvc/azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - azure-pipelines
```
Comment (allydurks) [inline QuoteSrvc/azure-pipelines.yml]:
```suggestion
    - master
```

File: OrderSrvc/azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - azure-pipelines
```
Comment (allydurks) [inline OrderSrvc/azure-pipelines.yml]:
```suggestion
    - master
```

File: DealerService/azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - azure-pipelines
```
Comment (allydurks) [inline DealerService/azure-pipelines.yml]:
```suggestion
    - master
```

File: CatalogSrvc/azure-pipelines.yml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+resources:
+- repo: self
+queue:
+  name: Hosted Ubuntu 1604
+  demands: java
+trigger:
+  branches:
+    include:
+    - azure-pipelines
```
Comment (allydurks) [inline CatalogSrvc/azure-pipelines.yml]:
```suggestion
    - master
```

**Review by allydurks — COMMENTED** [review]
These builds should be queued off master, `azure-pipelines` was the branch used for dev.

File: CatalogSrvc/azure-pipelines.yml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  displayName: 'gradlew build'
+  inputs:
+    gradleWrapperFile: CatalogSrvc/gradlew
+    workingDirectory: CatalogSrvc
+
+- task: Docker@1
+  displayName: 'Build an image'
+  inputs:
+    azureSubscriptionEndpoint: $(SERVICE_CONNECTION)
+    azureContainerRegistry: pumrpmicroacr.azurecr.io
```
Comment (dtzar) [inline CatalogSrvc/azure-pipelines.yml]: Let's put this ACR value to a variable throughout and then the rest looks good!

**Review by dtzar — CHANGES_REQUESTED** [review]

File: CatalogSrvc/azure-pipelines.yml (hunk 7 above)
Comment (allydurks) [reply CatalogSrvc/azure-pipelines.yml]: updated these to $(ACR_SERVER) and set that variable in the azure devops pipelines - in the same location as the $(SERVICE_CONNECTION) variable

**Review by dtzar — CHANGES_REQUESTED** [review]
One thing I realized is missing - the container image names are all the same.  They should conform to the same naming structure as the upstream docker hub images and readme.

i.e. for the [catalog image](https://hub.docker.com/r/microsoft/pumrp-catalog/)
`docker build -f /home/vsts/work/1/s/CatalogSrvc/Dockerfile -t pumrpmicroacr.azurecr.io/microsoft/partsunlimitedmrpmicro:645 /home/vsts/work/1/s`
--->
`docker build -f /home/vsts/work/1/s/CatalogSrvc/Dockerfile -t pumrpmicroacr.azurecr.io/microsoft/pumrp-catalog:645 /home/vsts/work/1/s`

**Review by dtzar — APPROVED** [review]

**Comment by dtzar** [comment]
Fixes #23

**Pull request merged**